src/pages/About.jsx

```diff
@@ -39,8 +40,12 @@ const developers = [
 function About() {
   return (
     <Container>
-      <h1 style={{ textAlign: 'center' }}>About UndocuScholar</h1>
-      <h2>Scholarship Search With UndocuScholar</h2>
+      <h1 style={{ textAlign: 'center' }}>
+        {`About ${BRAND_NAME}`}
```

Comment:
This actually doesn't look as clean as I thought 😦 . Maybe instead templates like this we can just use normal JSX but disable the one expression per line rule? In the `eslintConfig.rules` section you can add `"react/jsx-one-expression-per-line": "off"`

Reply:
This might even fit in one line with the `<h1>` tags